spring-social-zotero/src/main/java/org/springframework/social/zotero/api/impl/GroupsTemplate.java

```diff
@@ -350,4 +350,21 @@ public void deleteItem(String groupId, String citationKey, Long citationVersion)
         }
 
     }
+    
+    @Override
+    public ResponseEntity<String> deleteMultipleItems(String groupId, List<String> citationKeys, Long citationVersion) throws ZoteroConnectionException {
+        String citations = String.join(",", citationKeys);
+        String url = String.format("groups/%s/%s", groupId, "items?itemKey=" + citations);
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("If-Unmodified-Since-Version", citationVersion + "");
+
+        HttpEntity<JsonNode> dataHeader = new HttpEntity<JsonNode>(headers);
+
+        try {
+            return restTemplate.exchange(buildUri(url, false), HttpMethod.DELETE, dataHeader, new ParameterizedTypeReference<String>() {});
```

Comment:
you need to handle the response codes appropriately
https://www.zotero.org/support/dev/web_api/v3/write_requests#deleting_multiple_items